src/utils/voice_recorder.py

```
@@ -0,0 +1,187 @@
+
+        Args:
+            max_duration: 最長連接時間（秒）
+        """
+        try:
+            await asyncio.sleep(max_duration)
+            if self.is_connected:
+                await self.disconnect_from_voice()
+                logger.info(f"連接達到最長時間 {max_duration} 秒，自動斷開")
+        except asyncio.CancelledError:
+            pass
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)